cmds/boot/systemboot/main.go

```diff
@@ -31,6 +31,11 @@ var defaultBootsequence = [][]string{
 	{"localboot", "-grub"},
 }
 
+// VPD variable for enabling IPMI BMC overriding boot order, default is not set
+const VpdBmcBootOrderOverride = "bmc_bootorder_override"
+
+var bmcBootOverride bool = false
```

Comment:
the zero value of a bool is false, so you might consider dropping the = false part.

Reply:
done.